scionlab/tests/test_user_as_models.py

```diff
@@ -372,92 +427,150 @@ def _setup_vpn_attachment_point(self, ap):
         ap.save()
 
 
+def get_all_random_as_defs(has_vpn=False, seed=1) -> List[List[AttachmentPoint]]:
+    """
+    Generates a list of lists of AttachmentPoints.
+    For each ISD sample an AttachmentPoint as a list composed of a single element,
+    and if there are more than one AttachmentPoint in the said ISD, sample a list
+    of at least 2 elements
+    :param bool has_vpn: use only VPN capable AttachmentPoints
+    :param int seed:
+    :return List[List[AttachmentPoint]]:
```

Comment:
We cannot move `as_defs2aps` inside `get_all_random_as_defs` since the latter is needed to generate the parameterized tests, and the former returns true `AttachmentPoint`s objects, which we can only retrieve at test time when the fixtures are loaded. 
In fact, the function signature was still the one when I was using the fixture loading trick to generate parameterized tests with real objects.